### ISSUE TITLE
ValueError: need more than 0 values to unpack

### ISSUE THREAD

**Issue by davidmaxwaterman** [gpodder/gpodder#143]
GPodder invited me to report this problem, so I am. Here is the console output :

```
$ gpodder
1473588883.928676 [gpodder.sync] WARNING: Could not find gpod
1473588883.930683 [gpodder.sync] WARNING: Could not find eyed3.mp3
/usr/share/gpodder/extensions/ubuntu_unity.py:59: PyGIWarning: Unity was imported without specifying a version first. Use gi.require_version('Unity', '7.0') before import to ensure that the right version gets loaded.
  from gi.repository import Unity, GObject
1473589091.732538 [gpodder.my] WARNING: Flush requested, but sync disabled.
1473589485.250578 [gpodder.my] WARNING: Flush requested, but sync disabled.
1473589485.341347 [gpodder.log] ERROR: Uncaught exception: Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/main.py", line 2240, in on_after_update
    self.update_podcast_list_model(select_url=url)
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/main.py", line 2075, in update_podcast_list_model
    self.podcast_list_model.update_all()
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/model.py", line 732, in update_all
    self.update_by_iter(row.iter)
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/model.py", line 755, in update_by_iter
    zip(*[c.get_statistics() for c in channels]))
ValueError: need more than 0 values to unpack

Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/main.py", line 2240, in on_after_update
    self.update_podcast_list_model(select_url=url)
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/main.py", line 2075, in update_podcast_list_model
    self.podcast_list_model.update_all()
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/model.py", line 732, in update_all
    self.update_by_iter(row.iter)
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/model.py", line 755, in update_by_iter
    zip(*[c.get_statistics() for c in channels]))
ValueError: need more than 0 values to unpack
1473589497.504817 [gpodder.gtkui.main] ERROR: Exception happened while updating download list.
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/main.py", line 1163, in update_downloads_list
    self.update_podcast_list_model(channel_urls)
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/main.py", line 2080, in update_podcast_list_model
    self.podcast_list_model.update_sections()
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/model.py", line 737, in update_sections
    self.update_by_iter(row.iter)
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/model.py", line 755, in update_by_iter
    zip(*[c.get_statistics() for c in channels]))
ValueError: need more than 0 values to unpack
1473589583.892281 [gpodder.log] ERROR: Uncaught exception: Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/main.py", line 2426, in update_progress
    self.update_podcast_list_model([channel.url])
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/main.py", line 2080, in update_podcast_list_model
    self.podcast_list_model.update_sections()
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/model.py", line 737, in update_sections
    self.update_by_iter(row.iter)
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/model.py", line 755, in update_by_iter
    zip(*[c.get_statistics() for c in channels]))
ValueError: need more than 0 values to unpack

Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/main.py", line 2426, in update_progress
    self.update_podcast_list_model([channel.url])
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/main.py", line 2080, in update_podcast_list_model
    self.podcast_list_model.update_sections()
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/model.py", line 737, in update_sections
    self.update_by_iter(row.iter)
  File "/usr/lib/python2.7/dist-packages/gpodder/gtkui/model.py", line 755, in update_by_iter
    zip(*[c.get_statistics() for c in channels]))
ValueError: need more than 0 values to unpack
```

```
ii  gpodder        3.9.0-2
$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04.1 LTS"
```

**Comment by thp** [comment]
For which podcast / subscription does this happen? Does it happen always? If so, are there any other errors (e.g. when loading the podcast list) before? Can you reproduce the issue with gPodder 3.9.1 from the Ubuntu PPA?

**Comment by elelay** [comment]
I've seen this in the gtk3 branch when emptying a section or when no subscription at all.

My fix is check for empty: https://github.com/elelay/gpodder/blob/gtk3-new/src/gpodder/gtkui/model.py#L757

**Comment by opello** [comment]
I saw this today on Windows 10 Version 1511 (build 10586.633) after updating from 3.9.0 to 3.9.1, both distributions from [PortableApps](http://portableapps.com/apps/internet/gpodder_portable).  I updated because I'd been out of date and had just added a few new podcasts via gpodder.net and they were not being picked up in 3.9.0 despite what I did (restarted, checked for new episodes, etc.).

The new podcasts I'd just confirmed getting data for were:
```
http://feeds.waywordradio.org/awwwpodcast
http://www.npr.org/rss/podcast.php?id=510303
http://rss.acast.com/nature
http://feeds.feedburner.com/SlateAmicusWithDahliaLithwick
http://feeds.gimletmedia.com/hearreplyall
https://changelog.com/podcast/feed
http://historyofenglishpodcast.com/feed/podcast/
http://feeds.soundcloud.com/users/soundcloud%3Ausers%3A16745745/sounds.rss
```
But I'm not sure which triggered the problem.

Here is a lightly redacted log of the exception I received:
```
2016-10-28 14:22:58,994 [gpodder.gtkui.main] ERROR: Exception happened while updating download list.
Traceback (most recent call last):
  File "<snip>\gPodderPortable\App\gPodder\src\gpodder\gtkui\main.py", line 1160, in update_downloads_list
    self.update_podcast_list_model(channel_urls)
  File "<snip>\gPodderPortable\App\gPodder\src\gpodder\gtkui\main.py", line 2084, in update_podcast_list_model
    self.podcast_list_model.set_channels(self.db, self.config, self.channels)
  File "<snip>\gPodderPortable\App\gPodder\src\gpodder\gtkui\model.py", line 692, in set_channels
    self.update_by_iter(it)
  File "<snip>\gPodderPortable\App\gPodder\src\gpodder\gtkui\model.py", line 755, in update_by_iter
    zip(*[c.get_statistics() for c in channels]))
ValueError: need more than 0 values to unpack
```

**Comment by elelay** [comment]
The gtk3 branch has been released as 3.10.0 so the bug is fixed.
